src/Modalize.tsx

```diff
@@ -460,7 +460,6 @@ export default class Modalize extends React.Component<IProps, IState> {
 
           <KeyboardAvoidingView
             behavior="position"
-            style={{ paddingBottom: adjustToContentHeight ? 0 : this.handleHeight }}
```

Comment:
We need this. If you add an input at the bottom on the `DefaultContent` example and you display the keyboard, this padding is added to move the scrollview up and align it with the keyboard top position. 

Even thought, when I added this line back the snapping list was aligned at the bottom like expected.

Can you confirm this? If you still have the issue, we should an extra flag to check if this padding is needed or not

Reply:
Alright, you are right, I just tried now with the footer component. We definitely need a check with header/footer components to handle this padding or not

Reply:
Yes, I have this issue in the example project on a iPhone X simulator